### DISCUSSION TITLE
Weird behaviours after crash

### DISCUSSION THREAD

**Discussion by klaudius666** [iNavFlight/inav#7839] [Q&A]
**PLEASE MAKE SURE YOU READ AND UNDERSTAND THE SOCIAL MEDIA SUPPORT CHANNELS. QUESTIONS ABOUT FLASHING, CONFIGURING, PILOTING MAY BE CLOSED WITHOUT FURTHER INTERACTION.**

* [Telegram channel](https://t.me/INAVFlight)
* [Facebook group](https://www.facebook.com/groups/INAVOfficial)
* [RC Groups thread](https://www.rcgroups.com/forums/showthread.php?2495732-Cleanflight-iNav-%28navigation-rewrite%29-project)

**Please double-check that nobody reported the issue before by using search in this bug tracker.**

**PLEASE DELETE THE TEXT ABOVE AFTER READING AND UNDERSTANDING IT**

****

## Steps to Reproduce

1. Crashed from 10 meters
2.arming by accicent when drone was upside down
3.disconnect battery

4. Lay the quad flat. Connect battery
Always  Accro mode 

When arming the quad... It goes up but no roll, no pitch, no yaw... Disarming
Another arming.... The quad flip over brutally.
Disaminng... Rearming....  Nothing..

Go back.to home... Test again... Same problem.
Test motors via configurator.. working ok

Flash beta flight... Put initial propellers used during crash.... Flying good...

C ant understand.....

The fact was my black box was full and had to clear it. Even after flashing betaflight
 gps and magnetometer from m8q marek

 
Flashed beta flight....

.

## Additional context

---

- FC Board name and vendor: mambaF722i2c
- INAV version string: 3.0

**Comment by DzikuVx** [comment]
impossible to deternmine the cause without the blackbox log of the event
